src/forms/yup.js

```diff
@@ -19,16 +19,27 @@ import * as yup from 'yup';
 
 const ARRAY_INDEX_REGEX = /\[([^)]+)\]/;
 
+const isCountrySelected = value => {
+  if (!value || value === '') {
+    return false;
+  }
+  return true;
+};
+
+yup.addMethod(yup.string, 'selected', function () {
+  return this.test(
+    'selected',
+    params => ({ key: 'form.validation_select_field_required', params }),
+    value => isCountrySelected(value)
+  );
+});
```

Comment:
@josebui would it make sense to use an arrow function here?

Reply:
@paulschreiber you mean instead of `'selected', function () {`? If you see there yup expects the use of `this` inside it, so we have to use `function() {}` to be able to use that binded this